src/styles/colour/index.md.njk

```diff
@@ -0,0 +1,19 @@
+---
+title: Colour
+description:
+section: Styles
+aliases:
+layout: layout-styles.njk
+---
+
+{% from "_example.njk" import example %}
+
+Always use the GOV.UK colour palette.
+
+## Colour contrast
+The colour contrast ratio for text and interactive elements should be at least 4.5:1 as recommended by the W3C. Test your service to meet this standard.
+
+## Sass variables
+Use the Sass variables listed below, so you'll get any updates to the GOV.UK palette when you update GOV.UK Frontend.
+
+{{ example({group: 'styles', item: 'colour', example: 'default'}) }}
```

Comment:
This page is a bit of an odd one out, it's not really an "example" as such so should we just put the contents of the example in the page itself?

Only negative being is that the example container simulates the breakpoint change when the column gets smaller - Putting it in the page itself may require you to break the table at a larger / custom breakpoint.

Reply:
I think I'd agree - there's no need for the user to see it in isolation or to see the code, so I think it'd make sense just to make it part of the page.

Reply:
think I agree, however we might run into the whole markdown indentation issue if I move it into the main page, I'll have a bash

Reply:
so I've done this and yeh the breakpoints aren't working for me any more. Even if I use 'from: desktop' instead of tablet, the mobile view doesn't kick in fast enough and the table gets all squashed up. We don't seem to have a larger breakpoint.

As an aside, our desktop breakpoint is 769px which seems very small

Reply:
Yeah, the examples container was simulating "container queries" so it cares about the width of the thing as opposed to the width of the entire screen (which includes the fixed LH nav).

We haven't changed any of the breakpoints. 769px will just be 1px above 768px (standard iPad portrait) so that would be fairly standard for a small desktop view to kick in (and because we usually limit the content with the old `site-width-container` that was only a couple of hundred px away from as big as it gets). 

As this is a custom layout for the colour page I'd think it'd be ok to add another class name to the item and then use that to define a new media query to switch to the desktop layout only on screens bigger than 1024px for that item only?